src/js/containers/recipient/RecipientContainer.jsx

```diff
@@ -34,18 +36,27 @@ export class RecipientContainer extends React.Component {
         };
 
         this.request = null;
+        this.updateSelectedFy = this.updateSelectedFy.bind(this);
     }
 
     componentDidMount() {
+        if (!Object.keys(this.props.params).includes('fy')) {
```

Comment:
Hopefully when we migrate to React Router we can migrate this logic into a default route or a Redirect.